components/shared/upload/DropUpload.vue

```diff
@@ -0,0 +1,190 @@
+                }}</span>
+                <span class="has-text-left"
+                  ><b>{{ $t('mint.collection.formats') }}</b> {{ format }}</span
+                >
+              </div>
+              <NeoIcon
+                v-if="!file && !url"
+                :icon="icon"
+                class="cloud-icon"
+                size="is-large"
+                icon-pack="fas" />
+              <div v-if="url && !isModelMedia" @click.prevent>
+                <NeoIcon icon="circle-check" class="check-icon" />
+              </div>
+              <NeoIcon v-if="hasError" icon="eye-slash" size="is-large" />
+
+              <p v-if="!file" class="description mt-6">
+                {{ label }}
+              </p>
+              <div v-else class="is-flex is-align-items-center mt-6">
+                <span class="description mr-5">
+                  {{ file.name }} {{ $t('uploaded') }}
+                </span>
+                <div class="upload-file-label">
+                  {{ $t('chooseFile') }}
+                </div>
+              </div>
+            </div>
+          </section>
+        </o-upload>
+      </b-field>
+      <transition v-if="fileSizeFailed" name="fade">
+        <div class="help is-danger">
+          {{ $t('tooltip.failedMaxSize') }}
+        </div>
+      </transition>
+      <transition v-if="checkFailed" name="fade">
+        <div class="help is-danger">
+          {{ $t('tooltip.needToUploadNFTFile') }}
+        </div>
+      </transition>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { MAX_UPLOADED_FILE_SIZE } from '@/utils/constants'
+import { OUpload } from '@oruga-ui/oruga'
+import { NeoIcon } from '@kodadot1/brick'
+
+const MediaResolver = defineAsyncComponent(
+  () => import('@/components/media/MediaResolver.vue')
+)
+
+const props = withDefaults(
+  defineProps<{
+    label?: string
+    icon?: string
+    preview?: boolean
+    expanded?: boolean
+    accept?: string
+    format?: string
+  }>(),
+  {
+    label:
+      'Drop your NFT here or click to upload or simply paste image from clipboard',
+    icon: 'cloud-arrow-up',
+    preview: false,
+    expanded: false,
+    accept: '',
+    format:
+      'BMP, GIF, JPEG, PNG, SVG, TIFF, WEBP, MP4, OGV, QUICKTIME, WEBM, GLB, FLAC, MP3, JSON',
+  }
+)
+
+const file: any = ref(null)
+const url = ref('')
+let hasError = ref(false)
+let checkFailed = ref(false)
+let fileSizeFailed = ref(false)
+
+const fileSizeLimit = MAX_UPLOADED_FILE_SIZE
+
+const emit = defineEmits(['input'])
+
+const mimeType = computed(() => {
+  if (file?.value?.type) {
+    return file?.value.type
+  }
+
+  const fileExtension = file?.value?.name.split('.').pop() || ''
+  const extensionToMimeType = {
+    glb: 'model/gltf-binary',
+  }
+  if (fileExtension in extensionToMimeType) {
+    return extensionToMimeType[fileExtension]
+  }
+  return ''
+})
+
+const isModelMedia = computed(() => {
+  return mimeType.value.startsWith('model')
+})
```

Comment:
not relevant for zip files

Reply:
so along with zip files it should also support these right for description , since its a common component? 
<img width="619" alt="Screen Shot 2023-03-21 at 8 44 46 PM" src="https://user-images.githubusercontent.com/39299315/226796810-28105bbc-843a-4fc5-bc99-371eb7efe9d1.png">

Reply:
sorry, you are right. it is a shared component
some work will be required to adept it to zip files
or maybe make a zip files specific version of it